Tests/js/views/fields/assets/ez-relation-editview-tests.js

```diff
@@ -0,0 +1,419 @@
+            });
+
+            this.view = new Y.eZ.RelationEditView({
+                container: '.container',
+                field: this.field,
+                fieldDefinition: this.fieldDefinition,
+                content: this.content,
+                version: this.version,
+                contentType: this.contentType,
+                destinationContent: this.destinationContent,
+                destinationContentId: null,
+            });
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+        },
+
+        _testAvailableVariables: function (required, expectRequired) {
+            var fieldDefinition = this._getFieldDefinition(required),
+                that = this;
+
+            this.view.set('fieldDefinition', fieldDefinition);
+
+            this.view.template = function (variables) {
+                Y.Assert.isObject(variables, "The template should receive some variables");
+                Y.Assert.areEqual(9, Y.Object.keys(variables).length, "The template should receive 9 variables");
+                Y.Assert.areSame(
+                    that.jsonContent, variables.content,
+                    "The content should be available in the field edit view template"
+                );
+                Y.Assert.areSame(
+                    that.jsonVersion, variables.version,
+                    "The version should be available in the field edit view template"
+                );
+                Y.Assert.areSame(
+                    that.jsonContentType, variables.contentType,
+                    "The contentType should be available in the field edit view template"
+                );
+                Y.Assert.areSame(
+                    fieldDefinition, variables.fieldDefinition,
+                    "The fieldDefinition should be available in the field edit view template"
+                );
+                Y.Assert.areSame(
+                    that.field, variables.field,
+                    "The field should be available in the field edit view template"
+                );
+
+                Y.Assert.areSame(expectRequired, variables.isRequired);
+                return '';
+            };
+            this.view.render();
+        },
+
+        "Test not required field": function () {
+            this._testAvailableVariables(false, false);
+        },
+
+        "Test required field": function () {
+            this._testAvailableVariables(true, true);
+        },
+
+        "Test validate no constraints": function () {
+            var fieldDefinition = this._getFieldDefinition(false);
+
+            this.view.set('fieldDefinition', fieldDefinition);
+            this.view.render();
+            this.field = {fieldValue: {destinationContentId: 45}};
+
+            this.view.validate();
+            Y.Assert.isTrue(
+                this.view.isValid(),
+                "An empty relation is valid"
+            );
+            Y.Assert.areSame(
```

Comment:
this assertion has nothing to do with the purpose of the test. But indeed you should have a test called `Should fill the destinationContentId attribute from the field` with this assertion.